### PR TITLE
Doc fix folder name

### PR DESCRIPTION
**Pull request by akanex115**
## Related PRs
- _None_

## Checklist (Developers)
- [x] `Related PRs` is correctly modified, if it is not `None`
- [x] One of the followings is handled
  - At least one or more issue are linked to this PR under `development`
  - [The issue template](https://github.com/ajktown/.github/blob/main/issue_template.md) is directly copied above the `Related PRs`
    - You can directly copy background with issue from the samed/different repository.
  - One of the related PRs is linked to the issue under `development`
- [x] Assignee is set
- [x] Labels are set
- [x] Title is checked
- [x] `yarn inspect` is run
- [x] `TODOs` of associated issue (or TODOs here, if present) are handled and checked
- [x] Final Operation Check is done

## Checklist (Reviewers)
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Every item on the checklist has been addressed accordingly

### CODE REVIEW

File: dev/sequence_diagrams/on-click-post-word-from-undo.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+```plantuml
```
Comment (mlajkim) [inline dev/sequence_diagrams/on-click-post-word-from-undo.md:1]: Make sure the changes you made here is applied in the PR https://github.com/ajktown/docs/pull/33

Comment (mlajkim) [reply dev/sequence_diagrams/on-click-post-word-from-undo.md:1]: Merge Order 
1. https://github.com/ajktown/docs/pull/33
2. https://github.com/ajktown/docs/pull/35

**Review by mlajkim — COMMENTED** [review]
First review

**Comment by mlajkim** [comment]
## TODOs
- [ ] Actually make the directory name `diagrams`

**Comment by akanex115** [comment]
Handled in https://github.com/ajktown/docs/pull/37

**Pull request closed without merging**